common/etc/opx/startup/config-interfaces

```diff
@@ -14,7 +14,8 @@
 ##		- opx-config-hybrid-group (Z9264F-ON only)
 #   EXAMPLES:
 #   (On a Dell EMC S6010-ON)
-#	opx-config-fanout e101-001-0 4x1 10g
+#	opx-config-fanout --port e101-001-0 --mode 4x1 --speed 10g
+#	opx-config-fanout --pg_name port-group-7 --mode 4x4 --speed 10g
```

Comment:
This isn't right. The port-group style fanout applies to the Dell EMC S5296F-ON and S5248F-ON. Please make that clarification. Thanks

Reply:
Done